packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-card.tsx

```diff
@@ -1,10 +1,10 @@
-import React from 'react';
+import { Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
 import classNames from 'classnames';
-import { Icon, Text, Popover } from '@deriv/components';
-import { Localize, localize } from '@deriv/translations';
+import React from 'react';
```

Comment:
Please check the import rules https://github.com/binary-com/deriv-app/blob/master/docs/JavaScript/README.md#import-rules